assets/src/apps/authoring/components/PropertyEditor/custom/CustomButtonTemplate.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { CSSProperties, useRef, useState } from 'react';
+interface CustomButtonProps {
+  label: string;
+  onChange: any;
+}
+const CustomButtonTemplate: React.FC<CustomButtonProps> = (props: any) => {
```

Comment:
can we name this something like "EditJsonButtonTemplate" ? because I feel like this is a bit more specific than a button